packages/atlas/src/components/_auth/SignUpModal/SignUpModal.tsx

```diff
@@ -63,6 +63,7 @@ export const SignUpModal = () => {
   const setYppModalOpenName = useYppStore((state) => state.actions.setYppModalOpenName)
   const setYtResponseData = useYppStore((state) => state.actions.setYtResponseData)
   const { displaySnackbar } = useSnackbar()
+  const ytEmailIsValid = Boolean(ytResponseData?.email && !ytResponseData.email.includes('@plus.google.com'))
```

Comment:
I don't think we should consider any other email than google. 
 
```suggestion
  const ytEmailIsValid = Boolean(ytResponseData?.email && ytResponseData.email.endsWith('@google.com'))
```

Reply:
@WRadoslaw 
<img width="902" alt="image" src="https://github.com/Joystream/atlas/assets/13239251/829f4902-9630-4597-8a66-618b31f13f11">